autogpt_platform/frontend/src/stories/Typography.stories.tsx

```diff
@@ -0,0 +1,177 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Text } from "@/components/_new/Text/Text";
+import { StoryCode } from "@/stories/helpers/StoryCode";
+
+const meta: Meta<typeof Text> = {
+  title: "Design System/ Tokens /Typography",
+  component: Text,
+  parameters: {
+    layout: "fullscreen",
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Text>;
+
+export function AllVariants() {
+  return (
+    <div className="space-y-12">
+      {/* Typography System Documentation */}
+      <div className="space-y-8">
+        <div>
+          <h1 className="mb-4 text-4xl font-bold text-zinc-800">
+            Typography System
+          </h1>
+          <p className="text-lg leading-relaxed text-zinc-600">
+            Our typography system uses two carefully selected fonts to create a
+            clear hierarchy and excellent readability across all interfaces.
+          </p>
+        </div>
+
+        <div className="grid gap-8 md:grid-cols-2">
+          <div>
+            <h2 className="mb-4 text-2xl font-semibold text-zinc-800">
+              Font Families
+            </h2>
+            <div className="space-y-4">
+              <div className="rounded-lg border border-gray-200 p-4">
+                <h3 className="mb-2 font-semibold text-zinc-800">
+                  <a
+                    href="https://fonts.google.com/specimen/Poppins"
+                    target="_blank"
+                    rel="noopener"
```

Comment:
Change to rel="noopener noreferrer" for security

Reply:
Note that Storybook is an internal tool mainly used for us, the team, to document our "future" design system. So I see the security risk here as less critical. Still good to add 🙌🏽